KalebIrwin-Power-signoff.md

```diff
@@ -0,0 +1,38 @@
+## Function of the subsystem 
+The power subsystem will provide the necessary energy for the components to operate. 
+
+## Constraints 
+1. Output three voltages 
+  12V to power the chosen microcontroller 
+  5V and 3V to power common sensors
+2. Runs continuously with minimal user interaction 
+3. Designed modularly
```

Comment:
Not clear. Specify that the user may select one (or both?) of these when constructing the system.

Be more specific about ac input. For example, the Boeing 777 has 3-phase, 4-wire, 400 Hz, 115 V (phase voltage) power system ... that's "ac" but probably not what you meant!